cardano-db-sync/app/cardano-db-sync.hs

```diff
@@ -228,6 +229,15 @@ pSlotNo =
           <> Opt.metavar "WORD"
       )
 
+pKeepTxMetadata :: Parser [Text]
```

Comment:
A simple list may not be enough to represent this. For example does the emply list mean to keep all or to keep nothing?